Voice of Time/Transfer/ClientSocket.cs

```diff
@@ -54,8 +54,8 @@ long ID
         private readonly SemaphoreSlim itemInQueue = new(0, 1);
 
         /// <summary>
-        /// <para>A helping register witch holds the Callback Funcktions of <see cref="QueueItem"/>.</para>
-        /// This is used since the use of a Full-Duplex connection, witch no longer garantues that a anserw to a request is the next thing to expect.
+        /// <para>A helping register which holds the Callback Funcktions of <see cref="QueueItem"/>.</para>
+        /// This is used since the use of a Full-Duplex connection, which no longer garantues that a anserw to a request is the next thing to expect.
```

Comment:
ich versteh den satz nich :(

Reply:
Eine Map (Key->Value) für lambda funktionen. Der schlüssel stellt dabei die packageID dar (ID die anzeigt ob ein packet eine antwort ist oder nicht [ client -12-> server; server -12> client; client-> key=12  ])
Das wurde eingeführt da nicht mehr auf die antwort nach dem senden der frage gewerted wird sonder irgendwan in der zukunpft von einem anderen Process abgearbeitet wird